### ISSUE TITLE
Windows smartscreen

### ISSUE THREAD

**Issue by ebretteville** [sabrogden/Ditto#188]
Hye,
As there is no editor, ditto installer is blocked by the windows smartscreen...

Any thoughts?

**Comment by WeedCandy** [comment]
The code isn't signed, so Smartscreen is going to trigger. You can still install by clicking "Run Anyway". It would be nice to have a signed version, or at least hashes published on the website to mitigate man-in-the-middle attacks.

**Comment by ebretteville** [comment]
I can't, maybe my organisation restriction.

**Comment by WeedCandy** [comment]
If it's a work computer, then yes, it's likely your org prohibits the installation of unsigned code. You can ask for an exception to be made. If you have admin on your box, try enabling Developer settings and installing again.

**Comment by sabrogden** [comment]
Are you allowed to install apps from the windows store, if so try Ditto from there, https://www.microsoft.com/en-us/p/ditto-clipboard/9nblggh3zbjq#activetab=pivot:overviewtab

**Comment by ebretteville** [comment]
I tried both solution, dev mode can't be activated. Tried the link, the store opened and blacket out saying that it's disable by admin...
